.yarn/versions/4c05b7ba.yml

```diff
@@ -0,0 +1,41 @@
+releases:
+  "@radix-ui/react-accessible-icon": patch
```

Comment:
I wonder about all these if we should decline them all here, because we don't want 0.1.1 or whether this is ok because there's another commit asking for a minor, therefore it takes the highest one?

Reply:
AFAIK it is okay because the release will pick the highest version and roll with that.